src/main/java/net/dv8tion/jda/api/events/guild/update/GuildUpdateSplashEvent.java

```diff
@@ -60,6 +61,22 @@ public String getOldSplashUrl()
         return previous == null ? null : String.format(Guild.SPLASH_URL, guild.getId(), previous);
     }
 
+    /**
+     * Returns an {@link ImageProxy} for this guild's old splash image
+     * <p>
+     * <b>Note:</b> the old splash may not always be downloadable as it might have been removed from Discord
```

Comment:
```suggestion
     * Returns an {@link ImageProxy} for this guild's old splash image.
     * <p>
     * <b>Note:</b> the old splash may not always be downloadable as it might have been removed from Discord.
```